specification/network/resource-manager/Microsoft.Network/stable/2023-05-01/firewallPolicy.json

```diff
@@ -1719,6 +1723,20 @@
         "modelAsString": true
       }
     },
+    "FirewallPolicyIntrusionDetectionProfileOptions": {
+      "type": "string",
+      "description": "Possible profile values.",
```

Comment:
This isn't really specific. Its better to add more context as to the meaning or usage, here and/or on the enum values